src/osemgrep/cli_interactive/Unit_framed_zipper.ml

```diff
@@ -148,3 +148,22 @@ let%expect_test "position" =
   [%expect {|
     [3 | 3, 4, (5)]
     absolute: 4, relative: 2 |}]
+
+let%expect_test "top_bottom" =
+  let str_of_bool b = if b then "T" else "F" in
+  let zipper = FZ.of_list 3 [ 1; 2; 3 ] in
+  Common.pr2 (str_of_bool (FZ.is_top zipper));
```

Comment:
Honestly I prefer assert FZ.is_top_zipper than those pr2 str_of_bool stuff.
Not sure this is a good use case for ppx_expect tests.